zerver/models.py

```diff
@@ -1500,3 +1548,32 @@ class RealmAuditLog(models.Model):
     event_type = models.CharField(max_length=40) # type: Text
     event_time = models.DateTimeField() # type: datetime.datetime
     backfilled = models.BooleanField(default=False) # type: bool
+
+class CustomProfileField(models.Model):
+    realm = models.ForeignKey(Realm)  # type: Realm
+    name = models.CharField(max_length=100)  # type: Text
+    field_type = models.CharField(max_length=100)  # type: Text
```

Comment:
Should `field_type` be a PositiveSmallIntegerField (basically an enum) similar to Recipient.type?  I think we'll probably want to have a set of ~5 field types (number, short text, long text, etc.), and we want them to be well-defined types so that we can display different frontend UI widgets for each one.